src/travix/Travix.hx

```diff
@@ -107,7 +107,12 @@ class Travix {
       case 'java': new JavaCommand(cmd, args);
       case 'flash': new FlashCommand(cmd, args);
       case 'cpp': new CppCommand(cmd, args);
-      case 'php': new PhpCommand(cmd, args);
+      case 'php':
+        #if (haxe_ver < 4)
```

Comment:
Travix is primarily distributed with a precompiled neko binary, so that check will have to be done at runtime.